### PR TITLE
Configure circle ci

### PR DESCRIPTION
**Pull request by berrada-zouhair**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  deploy:
+    working_directory: ~/circleci-code-solidaire
+    machine:
+      enabled: true
+    steps:
+      - checkout
+      - run:
+          name: Deploy Master to Heroku
+          working_directory: backend
+          command: docker login --username=abid1991@hotmail.fr --password=eb6e5e1e-d611-4ecb-9921-4d2b0c1ccc6d registry.heroku.com
```
Comment (vdemeester) [inline .circleci/config.yml]: :scream: Use environment variables in circleci !!! not plain text password :sweat:

Comment (zeroual) [reply .circleci/config.yml]: Yeah. I want first to make it work . Next I will hide it.
I have a problem when I push the image to heroku registery 😅

**Comment by zeroual** [comment]
@RomainVernoux  @vdemeester plz , can you check this circleci configuration !

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+      - attach_workspace:
+          at: backend
+
+      - run: mkdir -p backend/target/classes/static/
+
+      - run: cp -a frontend/dist/frontend/. backend/target/classes/static/
+
+      - restore_cache:
+          key: maven-
```
Comment (vdemeester) [inline .circleci/config.yml:89]: Is this working ? (i.e. is this going to restore any cache ?)

Comment (vdemeester) [reply .circleci/config.yml:89]: shouldn't it be `maven-{{checksum …}}` ?

Comment (zeroual) [reply .circleci/config.yml:89]: i think that circleci will restore the last cache with key maven-
https://circleci.com/docs/2.0/caching/#source-caching

Comment (vdemeester) [reply .circleci/config.yml:89]: ok fair enough :wink:

**Review by vdemeester — APPROVED** [review]

**Pull request merged**